src/pages/costModels/costModel/updateMarkupDialog.tsx

```diff
@@ -168,33 +173,30 @@ class UpdateMarkupModelBase extends React.Component<Props, State> {
                         value="true" // '-'
                         onChange={this.handleSignChange}
                       />
-                    </div>
+                    </FlexItem>
                   </FormGroup>
-                </GridItem>
-                <GridItem lg={4} id="refMarkup">
-                  <FormGroup
-                    isInline
-                    fieldId="rate"
-                    label={t('rate')}
-                    helperTextInvalid={t('cost_models_wizard.markup.invalid_markup_text')}
-                  >
-                    <InputGroup style={styles.rateWidth}>
-                      <InputGroupText style={styles.sign}>{isDiscount ? '-' : '+'}</InputGroupText>
-                      <TextInput
-                        style={{ borderLeft: '0' }}
-                        type="text"
-                        aria-label={t('rate')}
-                        id="markup-input-box"
-                        value={this.state.markup}
-                        onKeyDown={this.handleOnKeyDown}
-                        onChange={this.handleMarkupDiscountChange}
-                        validated={this.markupValidator()}
-                      />
-                      <InputGroupText style={styles.percent}>%</InputGroupText>
-                    </InputGroup>
+                </Flex>
+                <Flex direction={{ default: 'column' }} alignSelf={{ default: 'alignSelfCenter' }}>
+                  <FormGroup fieldId="rate" helperTextInvalid={t('cost_models_wizard.markup.invalid_markup_text')}>
+                    <FlexItem style={styles.marginTopSix}>
```

Comment:
Instead of adding the same top margin to both radio and input text elements, add it once to their container.